src/proto/grpc/reflection/v1/reflection.proto

```diff
@@ -0,0 +1,147 @@
+// Copyright 2016 The gRPC Authors
```

Comment:
Are you planning to convert our reflection code to use v1 instead of v1alpha?  Are there any backward-compatibility problems with clients if we do that?

Reply:
v1 and v1alpha appear to have the same proto definition so it shouldn't be hard to migrate it to v1 but this is more like a mechanical synchronization between two proto directories.

Reply:
Are you planning to automate the mirroring of the grpc/grpc-proto repo into the src/proto/grpc tree in this repo?  If not, why do we need to add proto files that we're not actually using?

Reply:
Yeah, that's a long-term goal for this. There are a couple hairy issues to be resolved to get that level though but this way (keeping both similar as possible) is more easy to maintain two proto repos without having an automatic sync. Internally we need to moving toward using the `grpc-proto` repo only so they should become identical enough.

Reply:
And the next plain for this particular proto, we're going to deprecate v1alpha internally and remove v1alpha proto from both repos.

Reply:
Updating reflection to v1 needs cross-language coordination, since all languages on on v1alpha to my knowledge. Swapping to v1 will break tooling as the gRPC method name would be different (because of the proto package).

Reply:
Oh, interesting -- for some reason I thought I remembered that java was already on v1.

If all languages are still using v1alpha, then I agree that we need a cross-language plan to update this.  Maybe we can update the clients to try v1 and then fall back to v1alpha is v1 returns UNIMPLEMENTED, then add v1 to servers without removing v1alpha, and then after some period of time, we can remove v1alpha.

Anyway, if we want to pursue this, let's have a separate design for it.